requirements.txt

```diff
@@ -11,16 +11,18 @@ imageio~=2.19
 joblib~=1.0
 matplotlib>=3.3.0
 nibabel~=3.2
+onnx
 onnxruntime~=1.7
```

Comment:
Alright, building this PR off of #1297 has taken care of another big chunk of failing tests. There are only 2 left! 

The error I'm seeing is:

```python
    def _add_onnxscript_fn(
        model_bytes: bytes,
        custom_opsets: Mapping[str, int],
    ) -> bytes:
        """Insert model-included custom onnx-script function into ModelProto"""
        # TODO(titaiwang): remove this when onnx becomes dependency
        try:
            import onnx
        except ImportError as e:
>           raise errors.OnnxExporterError("Module onnx is not installed!") from e
E           torch.onnx.errors.OnnxExporterError: Module onnx is not installed!
```

`onnxruntime` is installed, but `onnx` is not. Is this a new requirement, then? Hrm...

Reply:
I've set the upperbound for `onnxruntime` to address the same issue we encountered over at SCT:

After fixing this issue, the tests now pass! (We're still getting a failing CI because of ReadTheDocs, which is still being built using Python 3.7. But, we can take care of that in #1297/#1298.)